templates/bin_setup.erb

```diff
@@ -7,8 +7,8 @@
 set -e
 
 # Set up Ruby dependencies via Bundler
-gem list bundler --installed > /dev/null || gem install bundler
-bundle install
+command -v bundle > /dev/null || gem install bundler
```

Comment:
I think it is related.

The `gem list bundler` line being changed here is to make sure Bundler gets installed. It looks like RubyGems has that functionality built-in (install a gem, only if it's missing), and the flag for it is `--conservative`.